tests/workflow/eden.workflow.tests.txt

```diff
@@ -158,8 +158,10 @@ eden.escript.test -testdata ../registry/testdata/ -test.run TestEdenScripts/regi
 eden.escript.test -testdata ../eclient/testdata/ -test.run TestEdenScripts/air-gapped-switch
 /bin/echo Eden 2 dockers test (26/{{$tests}})
 eden.escript.test -testdata ../docker/testdata/ -test.run TestEdenScripts/2dockers_test
-/bin/echo Eden 2 dockers test with app state detector (27/{{$tests}})
+/bin/echo Eden 2 dockers test with app state detector (27.1/{{$tests}})
 eden.escript.test -testdata ../app/testdata/ -test.run TestEdenScripts/2dockers_test
+/bin/echo Testing replacement of one application with another (27.2/{{$tests}})
+eden.escript.test -testdata ../app/testdata/ -test.run TestEdenScripts/app_replace_test
```

Comment:
Not a blocker, but just to mention: test located here will run only on EdenGCP workflow for now, as we have `{{ if or (eq $workflow "large") (eq $workflow "gcp") }}` above.

Reply:
and we need to add this tests to new workflows, because EVE is using them. I suggest to put this in[ user app](https://github.com/lf-edge/eden/blob/master/tests/workflow/user-apps.tests.txt)

Reply:
I already put this to [user-apps.tests.txt](https://github.com/lf-edge/eden/pull/891/files#diff-5cbd34664e6aee16a80e7af4e5000a60450a56aab2aea2055d67f7499db28e02)

Reply:
My bad, didn't see it :D

Reply:
@giggsoff I didn't realize that the test would be skipped in the old workflow, but since we are going to move to the new workflow soon, I'm going to keep it as it is.